cypress/Shared/EditMeasurePage.ts

```diff
@@ -94,6 +94,8 @@ export class EditMeasurePage {
     public static readonly successMessage = '[class="toast success"]'
     public static readonly measureReferenceTable = '[data-testid="measure-references-table-body"]'
     public static readonly measureReferenceDiscardChanges = '[data-testid="cancel-button"]'
+    public static readonly deleteReference = '[data-testid*="delete-measure-reference"]'
+    public static readonly editReference = '[data-testid*="edit-measure-reference"]'
```

Comment:
These selectors use a wildcard selector, and will only work with a single item on the table.
I needed to use this because the full data-testid values include a trailing uuid that we won't know.